### ISSUE TITLE
installation with npm fails

### ISSUE THREAD

**Issue by jfgodoy** [ariporad/modular-redux#1]
I get this message when a try to install the module.
Any idea?

```
npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/home/jorge/.nvm/versions/node/v4.2.1/bin/node" "/home/jorge/.nvm/versions/node/v4.2.1/bin/npm" "install" "modular-redux"
npm ERR! node v4.2.1
npm ERR! npm  v3.3.12

npm ERR! shasum check failed for /tmp/npm-9900-a540e711/registry.npmjs.org/modular-redux/-/modular-redux-1.2.0.tgz
npm ERR! Expected: 4169a908eda5f5db770ce29207630dd04b1689d6
npm ERR! Actual:   c7fae6df4ca185b47e2a8745f238bb552cf4e9c8
npm ERR! From:     https://registry.npmjs.org/modular-redux/-/modular-redux-1.2.0.tgz
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /datos/development/proyectos/proyectos_csw/webgis2/npm-debug.log
```

**Comment by ariporad** [comment]
No, that's strange. What happens if you delete your `node_modules` folder and try again?

**Comment by jfgodoy** [comment]
I tried in a different computer, in a fresh project, even after a `npm cache clean`, but I got the same error.

**Comment by ariporad** [comment]
@jfgodoy: I'm going to try republishing, to see if that fixes it.

**Comment by ariporad** [comment]
@jfgodoy: Ok, I just published 1.3.0 (Big release, capitalized the error messages, **_and_** added punctuation!), try again.

**Comment by jfgodoy** [comment]
now is ok.
Thank you!! :)
